src/UCX.jl

```diff
@@ -318,6 +336,35 @@ function UCXEndpoint(worker::UCXWorker, conn_request::UCXConnectionRequest)
     UCXEndpoint(worker, r_handle[])
 end
 
+function UCXEndpoint(worker::UCXWorker, addr::UCXAddress)
+    GC.@preserve addr begin
+        _UCXEndpoint(worker, addr.handle)
+    end
+end
+
+function UCXEndpoint(worker::UCXWorker, addr_buf::Vector{UInt8})
+    GC.@preserve addr_buf begin
+        addr = Base.unsafe_convert(Ptr{API.ucp_address_t}, pointer(addr_buf))
+        _UCXEndpoint(worker, addr)
+    end
+end
+
+function _UCXEndpoint(worker::UCXWorker, addr::Ptr{API.ucp_address_t})
```

Comment:
```suggestion
function UCXEndpoint(worker::UCXWorker, addr::Ptr{API.ucp_address_t})
```

Underscore is unnecessary?

Reply:
No since there are two function with `Ptr{Cvoid}` :/